go.mod

```diff
@@ -3,12 +3,19 @@ module sigs.k8s.io/kubebuilder/v3
 go 1.15
 
 require (
+	github.com/cloudflare/cfssl v1.5.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/markbates/pkger v0.17.1
 	github.com/onsi/ginkgo v1.15.0
 	github.com/onsi/gomega v1.10.5
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/tools v0.0.0-20201224043029-2b0845dc783e
+	// pin this version of yaml.v3, changing it impacts the test output
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
```

Comment:
Which import requires this? Just curious if we could somehow prevent using two different YAML parsers as we are using `sigs.k8s.io/yaml v1.2.0` internally.

Reply:
the kustomize fn framework uses this

Reply:
Are the output changes considerable or breaking? Or can we just update the test and remove this indirect require?

Reply:
They aren't breaking, but `go` doesn't seem to pick a consistent version unless this is specified and the flapping between versions creates a lot of confusion and churn.